Search/TextSearch.py

```diff
@@ -0,0 +1,54 @@
+"""
+	Search Engine
+	by Vincent Jeanselme
+	and Matthieu Clin
+	vincent.jeanselme@gmail.com
+"""
+
+from Search.AbstractSearchType import abstractSearchType
+from Search.AbstractResult import abstractResult
+import Utils.WordProcessing as wp
+from collections import Counter
+
+class firstTextResult(abstractResult):
+	"""
+	Prints the first result
+	"""
+
+	def display(self):
+		if len(res) != 0:
+			print(self.query + " has the words in common with " + self.res[0])
+		else:
+			print("No match in the current index")
+
+class textSearch(abstractSearchType):
+	"""
+	Searchs through a reverseindex in order to find the text which has the most
+	word in common with query
+	Query has to be text
+	"""
+
+	def computeResult(self):
+		"""
+		Returns the result of the comparsion
+		"""
+		words = wp.extractWords(self.query)
+		words = wp.stemming(words)
+		indexedQuery = Counter(words)
```

Comment:
It was my first idea however in order to let a total freedom to make the Search and compute the distance, I have thought that it would be better to separate them